bfrec

```diff
@@ -179,7 +191,13 @@ uefi_capsule_update()
     # order once the capsule flag is set in the 'OsIndications' variable
     # which is done by the printf command below.
     $run mkdir -p "$mount_efi/EFI/UpdateCapsule" 2>/dev/null
-    $run cp $capsule_file "$mount_efi/EFI/UpdateCapsule/$capsule_efi"
+    if [ $policy = single ]; then
```

Comment:
I beleive we can implement the same logic as mlxbf-bootctl (line 254).

$run cp $capsule_file "$mount_efi/EFI/UpdateCapsule/${capsule_efi}1"
if [ $policy = "dual" ]; then
  $run cp $capsule_file "$mount_efi/EFI/UpdateCapsule/${capsule_efi}2"
fi

What do you think?

Reply:
Done